code/game/machinery/computer/card.dm

```diff
@@ -334,6 +335,18 @@
 					modify.update_virtual_wallet()
 			nanomanager.update_uis(src)
 
+		if ("age")
+			if (is_authenticated())
+				var/t2 = modify
+				if ((modify == t2 && (in_range(src, usr) || (istype(usr, /mob/living/silicon))) && istype(loc, /turf)))
+					var/temp_age = text2num(href_list["age"])
+					if(temp_age)
+						modify.age_id = trunc(abs((temp_age)))
```

Comment:
still permits very large numbers that can cause errors, like for example 999999999999999999999999999999999999999999999999999999999999999999999

just clamp it